Update tutorial - check answers (#991)
alphagov/govuk-prototype-kit: docs/documentation/make-first-prototype/show-users-answers.md

# Show the user’s answers on the ‘Check your answers’ page

> The Prototype Kit stores answers that users enter, and you can use them to make more realistic prototypes, for example by showing answers for users to check.

Reviewer: ```suggestion
The Prototype Kit stores answers that users have entered. Use these answers to make more realistic prototypes, for example by showing answers for users to check.
```

## Show the answer to question 1

1. Open `check-your-answers.html` in your `app/views` folder.
2. Find the `<dt>` tag that contains the text 'Name'.
3. Change 'Name' to 'Number of balls you can juggle'.

> 4. In the `<dd>` tag on the next line, change 'Sarah Philips' to `{{ data['how-many-balls'] }}`.

Reviewer: I know this line is probably just quoting what's in the UI, but isn't that surname usually spelled with two _l_s?

This is how we show data a user has entered – 'how-many-balls' is the `name` attribute from the `<input>` on the question page.

Update the screen reader text – change this
```
<span class="govuk-visually-hidden"> name</span>
```
to
```
<span class="govuk-visually-hidden"> number of balls you can juggle</span>
```

Screen readers will read this out but it will not appear on the page. Without this hidden text, screen reader users would just hear “Change” and not know what it’s for.

## Show the answer to question 2

1. Find the `<dt>` tag that contains the text 'Date of birth'.
2. Change 'Date of birth' to 'Your most impressive juggling trick'.
3. In the `<dd>` tag on the next line, change '5 January 1978' to `{{ data['most-impressive-trick'] }}`.

Change
```
<span class="govuk-visually-hidden"> date of birth</span>
```
to
```
<span class="govuk-visually-hidden"> your most impressive juggling trick</span>
```
Go to http://localhost:3000/start and answer the questions to check your answers show up correctly.

## Delete the remaining example answers

> There are example answers on the ‘Check your answers’ template page that you do not need.

Reviewer: Is is the example answers or the template page that the user does not need? If the former, then we could always reword this as "On the 'Check your answers' template page, there are example answers that you do not need."

Author: former and yes that sounds good will update
